packages/at_chops/lib/src/key/key_type.dart

```diff
@@ -1 +1,11 @@
-enum EncryptionKeyType { rsa2048, rsa4096, ecc, aes128, aes192, aes256 }
+enum EncryptionKeyType {
```

Comment:
This is going to be a blocker later...

Sometimes we want to infer an algo, but when we infer from a runtimeType we don't always have the key size, nor care about the key size.

But also, when we get the algo used to encrypt / sign from a string, it's useful to have the algo. It's questionable whether it's useful to have the size. Maybe we need to make this a sealed class later, but, then it's not open to extension...